cloudify_cli/cli/cfy.py

```diff
@@ -1133,6 +1158,14 @@ def __init__(self):
             help=helptexts.WITH_LOGS
         )
 
+        self.port = click.option(
+            '--port',
+            required=False,
+            type=int,
+            callback=validate_port,
```

Comment:
instead of the validate function, we can do `type=click.IntRange(0, 65535)` (i guess 1, 65534, because those are inclusive)
https://click.palletsprojects.com/en/6.x/api/#click.IntRange